INSTALLATION.md

```diff
@@ -126,6 +126,32 @@ sudo systemctl start pouch
 
 Afterwards, you can pull an image and run PouchContainer containers.
 
+## Run pouch command with non-root users
+
+To enable users other than root can run pouch command directly without `sudo`, pouchd creates a unix socket owned by pouch group. Users just need have pouch group on the machine and add user to pouch group.
+
+**1. create the pouch group**
+
+If you want to unix socket to be owned by pouch group, make sure it exist or create it.
```

Comment:
How about:
If you want the unix socket to be owned by pouch group, make sure it exists or you need to create it.